test/testPicture.cpp

```diff
@@ -108,8 +107,75 @@ TEST_CASE("Load PNG file", "[tvgPicture]")
 
     float w, h;
     REQUIRE(picture->size(&w, &h) == Result::Success);
+
+    REQUIRE(w == 1000);
+    REQUIRE(h == 1000);
```

Comment:
You can use Approx class for float point comparison.
(https://github.com/catchorg/Catch2/blob/devel/docs/assertions.md#floating-point-comparisons)

Reply:
I am not sure here. These float sizes are parsed back to int eg. for raw image data size. I would rather catch an error if even a slight change, as it might lead to an error. What do you think?

Reply:
@mmaciola agree on this.